app/src/oculusvr/cpp/DeviceDelegateOculusVR.cpp

```diff
@@ -23,14 +25,38 @@
 #include <vector>
 #include <cstdlib>
 #include <unistd.h>
+#include <VrApi_Types.h>
 
 #include "VrApi.h"
 #include "VrApi_Helpers.h"
 #include "VrApi_Input.h"
 #include "VrApi_SystemUtils.h"
 
+#include "VRBrowser.h"
+
 namespace crow {
 
+static ovrMatrix4f ovrMatrixFrom(const vrb::Matrix& aMatrix) {
```

Comment:
This looks like a good candidate for a member function in vrb::Matrix. Something like `CopyTransposed(float[4][4] aTarget)`? As a follow up if you agree.